scripts/fossa.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh | sudo bash
```

Comment:
is this really the only way to install the cli? and is the API key private? maybe we should make this a private ENV var in the travis build

Reply:
Right, had the same feeling. That's what is recommended in the docs, https://docs.fossa.com/docs/travisci#section-add-fossa-steps-to-travisyml.

About the API key, it's a "safe" push-only key associated to my account, and also the prescribed way.

Let me add notes in the script for context.

Reply:
fair enough, although let it be known that I think this is _super jank_

Reply:
![PureJank_Sleeve_web](https://user-images.githubusercontent.com/487433/62674470-62a84900-b957-11e9-876d-86c54b1ae2ca.jpg)